test/functional/page_objects/common_page.js

```diff
@@ -148,7 +148,7 @@ export function CommonPageProvider({ getService, getPageObjects }) {
 
               // Browsers don't show the ':port' if it's 80 or 443 so we have to
               // remove that part so we can get a match in the tests.
-              const navSuccessful = new RegExp(appUrl.replace(':80', '').replace(':443', '')
+              const navSuccessful = new RegExp(appUrl.replace(':80/', '/').replace(':443', '')
```

Comment:
I think we should make the same change here; `.replace(':443/', '/')` so that it doesn't incorrectly match a port starting with 443.

Reply:
done- made changes